src/main/java/ip/exception/BadDeadline.java

```diff
@@ -0,0 +1,23 @@
+package ip.exception;
+
+/**
+ * Exception thrown on encountering deadline issues.
+ */
+public class BadDeadline extends DukeException {
+    private String deadline;
+
+    public BadDeadline(String s) {
+        deadline = s;
+    }
+
+    @Override
+    public String toString() {
+        if (deadline.isEmpty()) {
+            return "Deadline not specified for event!\n"
+                   + "Correct example: `deadline Homework /by 2/5/2022 2359`";
```

Comment:
Might be better if the example is changed to DD/MM/YYYY 2359 or 31/8/2022 since the current example now, I can interpret it as DD/MM... or MM/DD making it abit confusing

Reply:
I agree! Or maybe another way would be to directly give instructions on formatting the date to users?